### ISSUE TITLE
Design spec seems not properly implemented

### ISSUE THREAD

**Issue by drindt** [ganfra/MaterialSpinner#10]
I've compared the appearance to the material design spec. There are some differences in padding and text sizes.
![material_design_compare](https://cloud.githubusercontent.com/assets/1647655/7671403/d20b9dd2-fcce-11e4-9fa3-153e49a3cfff.png)

**Comment by ganfra** [comment]
For the left padding, it's up to you to choose an adapter with no padding. If you need to align floating label, you can use the "ms_alignLabels" boolean attribute.
The thickness of the line may be change with the "ms_thickness" integer attribute.
For the space between floating label and text, I align it with MaterialEditText ...

**Comment by drindt** [comment]
Thank you for your quick reply. 
I played with ```ms_thickness``` and i would suggest supporting there "dp". I played with 1 and 2 as thickness but i had not the result which match the spec. Its to thin or to fat. :-)

It would also be nice to have options like the [MaterialEditText](https://github.com/rengwuxian/MaterialEditText) have for finer configuration of colors and behavior at all. Also the floating label there gets programmatically an alpha value. I am pretty sure you don't have to re-invent the wheel, the code can be adopted partially.
